demos/Shrine/Shrine/ShrineHeaderContentView.swift

```diff
@@ -108,7 +108,12 @@ class ShrineHeaderContentView: UIView, UIScrollViewDelegate {
 
   override func layoutSubviews() {
     super.layoutSubviews()
-
+    var safeAreaInset: CGFloat = 0
+    #if swift(>=3.2)
+      if #available(iOS 11.0, *) {
+        safeAreaInset = min(self.safeAreaInsets.top, 10)
```

Comment:
Why 10?

Reply:
the safeAreaInsets.top pushes it all too far down due to the existing 44 initially put, in landscape it is 0 hence should remain 0